src/harness/testcases/WINNF_FT_S_PAT_testcase.py

```diff
@@ -0,0 +1,124 @@
+              'antennaAzimuth': installationParam['antennaAzimuth'],
+              'antennaGain': installationParam['antennaGain'],
+              'antennaBeamwidth': installationParam['antennaBeamwidth'],
+              'cbsdCategory': device['cbsdCategory']}
+    return cbsd
+
+def fssdata(fss_record, rxAntennaGainRequired=None):
+    deploymentParam = fss_record['deploymentParam'][0]
+    installationParam = deploymentParam['installationParam']
+    if 'antennaAzimuth' not in installationParam:
+        installationParam['antennaAzimuth'] = ''
+        
+    fss = {'latitude': installationParam['latitude'],
+             'longitude': installationParam['longitude'],
+              'height': installationParam['height'],
+              'antennaAzimuth': installationParam['antennaAzimuth'],
+              'antennaGain': installationParam['antennaGain'],
+              'antennaDowntilt': installationParam['antennaDowntilt']}
+    if rxAntennaGainRequired is not None:
+        fss['rxAntennaGainRequired'] = rxAntennaGainRequired
+    return fss
+
+
+class PropAndAntennaModelTestcase(sas_testcase.SasTestCase):
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+      pass
+
+  def generate_FT_S_PAT_default_config(self, filename):
+    """Generates the WinnForum configuration for PAT.1."""
+    
+    # Load Devices
+    device_a = json.load(
+    open(os.path.join('testcases', 'testdata', 'device_a.json')))
+
+    # load FSS
+    fss_record_0 = json.load(
+      open(os.path.join('testcases', 'testdata', 'fss_record_0.json')))
+    fss_record_0['rxAntennaGainRequired'] = True
+    reliabilityLevel = -1
+    queries = {}
+    queries[0]= {'reliabilityLevel': reliabilityLevel, 'cbsd': cbsddata(device_a),'fss': fssdata(fss_record_0, True)}
+
+    # Load PPA
+    ppa_record = json.load(
+        open(os.path.join('testcases', 'testdata', 'ppa_record_3.json')))
+     
+    reliabilityLevel = -1
+    queries[1] = {'reliabilityLevel': reliabilityLevel, 'cbsd': cbsddata(device_a),'ppa': ppa_record['zone']['features'][0]}
+
+    # save file
+    with open(filename, "w") as outfile:
+        json.dump(queries, outfile, indent=4, separators=(',', ': '))
+      
+      
+  @configurable_testcase(generate_FT_S_PAT_default_config)
+  def test_WINNF_FT_S_PAT_1(self, config_filename):
+    """Query with CBSD and FSS/PPA
+   
+    If (rxAntennaGainRequired = True)
+    The response should have the pathlossDb, txAntennaGainDbi and rxAntennaGainDbi 
+ 
+    If (rxAntennaGainRequired = False or omitted)
+    The response should have the pathlossDb, txAntennaGainDbi
+    """
+    
+    propagationAntennaModel = QueryPropagationAntennaModel()
+
+    config = loadConfig(config_filename)
+    for configItem in  config:
+       testjson= json.dumps(config[configItem])
+
+       refResponse = propagationAntennaModel.computePropagationAntennaModel(testjson)
```

Comment:
For Kate: Shouldnt it be something like: 

`FakeSasAdmin().QueryPropagationAndAntennaModel(request)`